wallace/custom.py

```diff
@@ -786,44 +748,30 @@ def transmission():
     """
     exp = experiment(session)
 
-    # get the participant_id
-    try:
-        participant_id = request.values["participant_id"]
-        key = participant_id[0:5]
-    except:
-        exp.log("Error: /transmission request, participant_id not specified")
-        page = error_page(error_type="/transmission, participant_id not specified")
-        js = dumps({"status": "error", "html": page})
-        return Response(js, status=403, mimetype='application/json')
+    # get the key
+    key = participant_id[0:5]
```

Comment:
just an FYI, slicing will always assume the beginning of a string if the `start` part of the slice notation is omitted, so this is the same as saying `participant_id[:5]`